supabase/functions/oauth/index.ts

```diff
@@ -5,7 +5,7 @@ import { authURL } from "./auth-url.ts";
 import { encryptConfig } from "./encrypt-config.ts";
 import { corsHeaders } from "../_shared/cors.ts";
 
-console.log("Hello from Functions!");
+console.log("OAuth functions started.");
```

Comment:
Does not matter right now since we only have a single function but might help having specific start up logging in the future.